state-chain/runtime/src/chainflip/decompose_recompose.rs

```diff
@@ -283,36 +267,14 @@ mod tests {
 		assert_eq!(select_median::<u16>(vec![]), None);
 	}
 
+	#[test]
 	// For BTC, we witness multiple values, and median should be
```

Comment:
the comment is now inaccurate - but yeah, as mentioned, I think is better to just delete the test and inline the function

Reply:
I updated the test and the comment to it, so that it hopefully will make more sense.